ddtrace/llmobs/_llmobs.py

```diff
@@ -226,6 +228,30 @@ def disable(cls) -> None:
 
         log.debug("%s disabled", cls.__name__)
 
+    @classmethod
+    def annotation_context(cls, tags: Optional[Dict[str, Any]] = None) -> AnnotationContext:
+        """
+        Sets specified attributes on all LLMObs spans created while the returned AnnotationContext is active.
+        Do not use nested annotation contexts to override the same tags since the order in which annotations
+        are applied is non-deterministic.
+
+        :param tags: Dictionary of JSON serializable key-value tag pairs to set or update on the LLMObs span
+                     regarding the span's context.
+        """
+        return AnnotationContext(cls._instance.tracer, lambda span: cls.annotate(span, tags=tags))
+
+    @classmethod
+    def annotation_context_async(cls, tags: Optional[Dict[str, Any]] = None) -> AnnotationContextAsync:
```

Comment:
Does this need to be `async def ...`? Just to be safe, can we test out with an actual async example (an async openai call?)

Reply:
I don't think so, since the function is just meant to return the async context manager which will then be used by the user using `async with`

Will send you the code snippet i used to test offline!